### DISCUSSION TITLE
scalability Difference - number of connection between community and Entreprise Edition - Max node in cluster

### DISCUSSION THREAD

**Discussion by itweb62** [emqx/emqx#10360] [General]
Hello soory for asking this,

Does the community edition can scale as a cluster , there is a limitation of node ?
does a single node on a huge machine can handle 1 millions MQTT connexion ? for how many Vcore / Ram ?
Do i need a licence for scale more ?
I would buy the licence after my app have scale but i need this broker with open source for start my project. (lack of money)

Thank you.
Best regards

**Comment by ieQu1** [comment]
Hello,

Open-source edition has no limit on the number of connections and/or nodes in the cluster. But it doesn't include most of the plugins that enterprise edition supports (like Kafka bridge, database integrations, etc.)
I don't know much about the pricing for the Enterprise edition, you'd have to talk to the sales for this. Alternatively, you could try [EMQX serverless](https://www.emqx.com/en/cloud/serverless-mqtt) or [EMQX cloud](https://www.emqx.com/en/cloud/dedicated)
